RocketChat/Rocket.Chat.ReactNative
# Chore: Migrate i18n to Typescript

## Proposed changes


## Issue(s)	


## How to test or reproduce

- Check if the translations are work properly
- Go to Settings -> Language -> change between the languages

## Screenshots

## Types of changes




## Checklist



## Further comments
